engine/table/src/main/java/io/deephaven/engine/table/impl/sources/CharacterSparseArraySource.java

```diff
@@ -414,38 +414,39 @@ public void ensurePrevious(RowSet changedRows) {
             return;
         }
 
-        if (prevFlusher == null) {
-            return;
+        // Note that this method is used to precondition the column source before processing an update in parallel. So,
```

Comment:
Maybe we should rename the method. `ensurePreviousAndPreallocateBlocks` or some other such mouthful.